build.sbt

```diff
@@ -14,11 +16,17 @@ Test / testOptions +=
     "-minSuccessfulTests",
     "10000"
   )
+
 libraryDependencies ++= Seq(
-  "org.typelevel"     %% "cats-core"       % catsVersion,
-  "org.scalactic"     %% "scalactic"       % "3.2.13",
-  "org.scalatest"     %% "scalatest"       % "3.2.13"  % Test,
-  "org.scalatestplus" %% "scalacheck-1-14" % "3.1.2.0" % Test,
-  "org.scalacheck"    %% "scalacheck"      % "1.14.1"  % Test,
-  "org.typelevel"     %% "cats-effect"     % "3.3.12"
+  "org.typelevel"     %% "cats-core"           % catsVersion,
+  "org.scalactic"     %% "scalactic"           % "3.2.13",
+  "org.scalatest"     %% "scalatest"           % "3.2.13"  % Test,
+  "org.scalatestplus" %% "scalacheck-1-14"     % "3.1.2.0" % Test,
```

Comment:
Curious why you chose ScalaTest rather than MUnit

Reply:
I think I just used the setup from my ScalaCheck docs.  I'm not aware of the distinction/ramification.  (when I started this I only read the ScalaCheck docs and have never read docs concerning MUnit.)